scylla/tests/integration/session.rs

```diff
@@ -3225,16 +3232,32 @@ async fn test_vector_type_unprepared() {
 
     session
         .query_unpaged(
-            format!(
-                "INSERT INTO {}.t (a, b, c) VALUES (1, [1, 2, 3, 4], ['foo', 'bar'])",
-                ks
-            ),
-            &[],
+            format!("INSERT INTO {}.t (a, b, c) VALUES (?, ?, ?)", ks),
+            &(1, vec![1, 2, 3, 4], vec!["foo", "bar"]),
         )
         .await
         .unwrap();
 
-    // TODO: Implement and test SELECT statements and bind values (`?`)
+    let query_result = session
+        .query_unpaged(format!("SELECT * FROM {}.t", ks), &[])
+        .await
+        .unwrap();
```

Comment:
:question: Is there any reason why we're using `SELECT *` queries instead of specifying column names?